lib/rbvmomi/logging.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module RbVmomi
+  class << self
+    attr_writer :logger
+  end
+
+  def self.logger
+    require 'logger'
+    @logger ||= Logger.new(nil)
```

Comment:
NOTE my typical go to for the "default" logger is a nil logger, but maybe better to default to `$STDOUT` to maintain the same default behavior as before?

Reply:
Yeah I was thinking the same...probably should default to STDOUT